state_change_block/spec.json

```diff
@@ -0,0 +1,21 @@
+{
+    "Description": "Maintains a **state** and when **state** changes, a signal is notified that contains the `state` and `prev_state`.\nPersistence is used to maintain the **state**. If `use_persistence` is enabled, that value will populate the initial state and _not_ the `initial_state` configuration value. Any new states (from new groups) will use the configured `initial_state` for their initial state though",
+    "Properties": {
+        "backup_interval (seconds=600)": " Interval at which state is saved to disk.",
```

Comment:
Properties also have additional metadata (e.g., data type, default value, whether it allows expressions, etc). I think that information should probably also be included in `spec.json`.